modin/pandas/groupby.py

```diff
@@ -1045,26 +1041,30 @@ def _wrap_aggregation(
         DataFrame or Series
             Returns the same type as `self._df`.
         """
+        if not isinstance(numeric_only, NumericOnly):
+            numeric_only = NumericOnly(numeric_only)
```

Comment:
I wonder if we can unconditionally cast to `NumericOnly(numeric_only)`, even if it already is a proper enum...